### ISSUE TITLE
authwindow doesnt close

### ISSUE THREAD

**Issue by reggie-ahh** [seanfisher/angular-oauth1-client#6]
hi, 

whenever the user authorizes the app, the window doesnt close, i can see in the url that it gets the verifier and token, however no matter what callback url i use, it never closes the window. im trying this with ionic view (the app is an ionic app). i was hoping you could direct me on what i should be placing in my callback url, or if i should try the process in a proper emulator.

any help will be appreciated. 

thank you.

**Comment by Derrickbowen** [comment]
Yes, this must be done on a device or in an emulator. The auth window doesn't work in the desktop browser in the same way.

**Comment by reggie-ahh** [comment]
Even when i try it in an emulator, it still doesnt close the auth window. I have a feeling that it may be caused by the framework im using (ionic).

**Comment by Derrickbowen** [comment]
This is meant to be used with ionic. Have you installed the Cordova in app browser plugin?

**Comment by reggie-ahh** [comment]
Yes i have. Could it perhaps be caused by how wp-api v2 returns to the callback?

**Comment by seanfisher** [comment]
@reggie-ahh The code hasn't been tested with `WP-API v2` yet. I just pushed some changes as version [`v0.1.11`](https://github.com/seanfisher/angular-oauth1-client/tree/v0.1.11) if you'd like to see if that fixes your issue. If not, and you figure out the problem, please issue a pull request! Thanks :)

To answer your original question about the callback url, I added some more info to the README in the [Additional Information](https://github.com/seanfisher/angular-oauth1-client/tree/v0.1.11#additional-information) section.

**Comment by reggie-ahh** [comment]
yeah the window still isnt closing (tested with an android emulator). i also tried a new ionic install, same problem. maybe it could be because of wp-api v2? i'll keep trying to find the source of the problem.
